ros-cslam/src/pose_graph_builder/scripts/transform_listener_ros.py

```diff
@@ -39,7 +39,8 @@ def __init__(self):
         self.id_map = {}
         self.last_callback = rospy.get_time()
         self.optim_period = 0.5
-        self.optim_period_counter = -10.0
+        self.optim_period_counter = -5.0
+        self.n = 0
```

Comment:
Everything fine for me! But should we rename this `n` to something more explanatory, like `num_messages_received`? And we should add it to the docstring of the class as well.